tests/cupy_tests/statics_tests/test_correlation.py

```diff
@@ -6,4 +6,86 @@
 @testing.gpu
 class TestCorrelation(unittest.TestCase):
 
-    pass
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose()
+    def test_corrcoef(self, xp, dtype):
+        a = testing.shaped_arange((2, 3), xp, dtype)
+        return xp.corrcoef(a)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose()
+    def test_corrcoef_y(self, xp, dtype):
+        a = testing.shaped_arange((2, 3), xp, dtype)
+        y = testing.shaped_arange((2, 3), xp, dtype)
+        return xp.corrcoef(a, y=y)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose()
+    def test_corrcoef_rowvar(self, xp, dtype):
+        a = testing.shaped_arange((2, 3), xp, dtype)
+        y = testing.shaped_arange((2, 3), xp, dtype)
+        return xp.corrcoef(a, y=y, rowvar=False)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose()
+    def test_cov(self, xp, dtype):
+        a = testing.shaped_arange((2, 3), xp, dtype)
+        return xp.cov(a)
+
+    @testing.for_all_dtypes()
```

Comment:
Add `@testing.with_requires('numpy>=1.10')` because the `dtype` (probably accidentally) changed in numpy/numpy#4960.  While I regard `array([]).reshape(0, 0)` as a bug, let's reproduce the newer numpy's outputs.